### DISCUSSION TITLE
syntax highlighting for code chunk not working

### DISCUSSION THREAD

**Discussion by AmritKoirala2000** [MrGreensWorkshop/MrGreen-JekyllTheme#90] [Problems] [answered]
**Describe the bug**
 The syntax highlighting for code is not working even in the example blog post. 

**Screenshots**

**Expected behavior**

**To Reproduce**

**Desktop (please complete the following information):**

**Mobile devices (please complete the following information):**

**Additional context**

**Comment by MrGreensWorkshop** [comment, marked as answer]
Hi, there is no syntax highlighting feature. Check this page for all other features.
https://jekyll-theme-mrgreen-demo.mrgreensworkshop.com/posts/2022-02-10-Examples
